php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -734,6 +738,44 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
         }
     }
 
+    /**
+     * Parses the metadata of a LINST instrument and sets the proper variables
+     * in the instrument class accordingly
+     *
+     * This is intended to be overridden by a subclass of NDB_BVL_Instrument
+     * to add support for other file formats.
+     *
+     * @param string $filename The filename to be loaded, or a base64 encoded
+     *                         string.
+     *
+     * @return void
+     */
+    function loadInstrumentMetadata(string $filename): void
+    {
+        $fp = fopen($filename, "r");
+        while (($line = fgets($fp, 4096)) !== false) {
+            $pieces = preg_split("/{@}/", $line);
+            $this->LinstLines[] = $pieces;
+
+            $type = $pieces[0];
+
+            switch ($type) {
+            case 'test':
```

Comment:
should 'test' maybe be 'name' instead? i feel like test doesnt mean much without context here and i'm under the impression that 'test' is an older implementation that we're now replacing with 'instrument'? or even 'testname' would be better i think